cmd/snap-confine/mount-support.c

```diff
@@ -634,6 +662,7 @@ void sc_populate_mount_ns(struct sc_apparmor *apparmor, int snap_update_ns_fd,
 			{"/media", true},	// access to the users removable devices
 #endif				// MERGED_USR
 			{"/run/netns", true},	// access to the 'ip netns' network namespaces
+			{"/mnt",.is_optional = true},	// to support the legacy-mnt interface
```

Comment:
The interface name isn't legacy-mnt.

Reply:
Ah, leftover from the older name. I'll fix this according to the final interface arrangement.